src/components/Table.vue

```diff
@@ -115,6 +119,9 @@ export default {
     reloadTable() {
       this.$store.dispatch("cleanCandidates");
       this.$store.dispatch("retrieveCandidates", this.params);
+    },
+    reloadTable2() {
+      this.$store.dispatch("retrieveCandidates", this.params);
     }
```

Comment:
Why not combine `reloadTable` and `reloadTable2` functions?

```
    reloadTable(clean) {
      if clean {this.$store.dispatch("cleanCandidates");}
      this.$store.dispatch("retrieveCandidates", this.params);
    }
```

Reply:
I preferred to create a `cleanAndReloadTable()` method although it's redundant, a call with `reloadTable(true)`or `reloadTable(false)`doesnt tell you the inner working of the function when you read it, i find it to be confussing